i8_terminal/service_result/earning_list_result.py

```diff
@@ -0,0 +1,103 @@
+from typing import Any, List
+
+import pandas as pd
+import plotly.express as px
+from pandas import DataFrame
+from plotly.subplots import make_subplots
+
+from i8_terminal.service_result import ServiceResult
+from i8_terminal.common.formatting import color, get_formatter
+from i8_terminal.common.layout import format_df
+from i8_terminal.config import METRICS_METADATA_PATH
+
+
+class EarningsListResult(ServiceResult):
+    
+    def __init__(self, data: DataFrame, context: Any=None):
+        super().__init__(data, context)
+
+
+    def _format_df(self, df: DataFrame, target: str) -> DataFrame:
+        metadata = pd.read_csv(METRICS_METADATA_PATH, delimiter=";")
+        formatters = {}
+        col_names = {}
+
+        columns = df.columns.to_list()
+
+        for col in columns:
```

Comment:
This is good for now but I think eventually, we need to have something like `metadata_service` class with a few methods that given a metric name, it returns metadata or formatters associate to it.